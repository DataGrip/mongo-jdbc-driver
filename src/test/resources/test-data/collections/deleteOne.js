// before
db.col.insert({name: "value1", v: 1});
db.col.insert({name: "value2", v: 2});
db.col.insert({name: "value2", v: 3});
// command
db.col.deleteOne({name: "value2"});
// clear
db.col.drop();