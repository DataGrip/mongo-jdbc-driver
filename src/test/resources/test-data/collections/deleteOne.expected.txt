|map|
|{acknowledged=true, deletedCount=1}|